devtools/ci/dockerfiles/build-env/Dockerfile

```diff
@@ -1,2 +1,12 @@
 FROM golang:1.13.1-alpine
-RUN apk --no-cache add git nodejs yarn make postgresql-client
+
+ENV PGDATA=/var/lib/postgresql/data PGUSER=postgres DB_URL=postgresql://postgres@
+RUN apk --no-cache add git nodejs yarn make postgresql postgresql-contrib
```

Comment:
how does `postgresql postgresql-contrib` differ from the previous `postgresql-client`?

Reply:
`postgresql` depends on/installs the client (as we do still need it). Instead of only adding the client, we're now adding the server & client. The contrib is for the `pgcrypto` extension we need.